.github/workflows/publish.yml

```diff
@@ -0,0 +1,55 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - 
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - 
+        name: Get tags
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      -
+        name: Set Username/Repo and ImagePrefix as ENV vars
+        run: |
+          echo "USER_REPO"=$(echo "$GITHUB_REPOSITORY" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV && \
+          echo "IMAGE_PREFIX"=$(echo "ghcr.io/${{ github.repository_owner }}/queue-worker" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV
```

Comment:
`IMAGE_PREFIX` looks different to how we did this elsewhere? Can you make the example more consistent with faas-netes or faas?